src/server/game/Chat/Channels/Channel.h

```diff
@@ -269,19 +286,20 @@ class TC_GAME_API Channel
         }
 
         typedef std::map<ObjectGuid, PlayerInfo> PlayerContainer;
-        typedef GuidSet BannedContainer;
+        typedef GuidUnorderedSet BannedContainer;
```

Comment:
Well, BannedContainer is just for element checking, so the change to unordered can't hurt.

PlayerContainer on the other hand; being ordered means that when a player leaves, next owner is searched from the player with the lowest guid in the channel, and as discussed in the PR page, this is meant to only be a cleanup, not modify the behaviour, and making PlayerContainer unordered will change it.